polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -103,7 +109,13 @@ export function Item({
     </Box>
   );
 
-  const textMarkup = <span className={styles.Text}>{contentMarkup}</span>;
+  const textMarkup = (
+    <span className={styles.Text}>
+      <Text as="span" variant="bodyMd">
```

Comment:
```suggestion
      <Text
        as="span"
        variant="bodyMd"
        fontWeight={active ? 'semibold' : 'regular'}
      >
```
We need a `font-weight` override for `active` items when checking Chromatic diff.
Line 65 in `ActionList.module.css` can be removed where `font-weight` is set to semibold for active items with this suggestion.

![25-22-fhbw3-o076f](https://github.com/Shopify/polaris/assets/26749317/4c1c68fa-5cc8-463e-bba7-08c87e9d6778)